packages/kbn-securitysolution-data-table/mock/test_providers.tsx

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/* eslint-disable @kbn/eslint/module_migration */
+
+import { euiDarkVars } from '@kbn/ui-theme';
+import { I18nProvider } from '@kbn/i18n-react';
+
+import React from 'react';
+import type { DropResult, ResponderProvided } from 'react-beautiful-dnd';
+import { DragDropContext } from 'react-beautiful-dnd';
+import { Provider as ReduxStoreProvider } from 'react-redux';
+import type { Store } from 'redux';
+import { BehaviorSubject } from 'rxjs';
+import { ThemeProvider } from 'styled-components';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { createStore as createReduxStore } from 'redux';
+
+import type { Action } from '@kbn/ui-actions-plugin/public';
+import { CellActionsProvider } from '@kbn/cell-actions';
+import { mockGlobalState } from './global_state';
+import { createKibanaContextProviderMock, createStartServicesMock } from '../lib/kibana_react.mock';
+import { localStorageMock } from './mock_local_storage';
+
+interface Props {
+  children?: React.ReactNode;
+  store?: Store;
+  onDragEnd?: (result: DropResult, provided: ResponderProvided) => void;
+  cellActions?: Action[];
+}
+
+export const kibanaObservable = new BehaviorSubject(createStartServicesMock());
+
+Object.defineProperty(window, 'localStorage', {
+  value: localStorageMock(),
+});
+window.scrollTo = jest.fn();
+const MockKibanaContextProvider = createKibanaContextProviderMock();
```

Comment:
Do we really use the `KibanaContext` in the component? I am not able to find where it's used. 
If it is needed maybe we should explain this requirement it in the package README, and what services it uses.

Reply:
Thanks for a heads up, I will look into it!

Reply:
removed